fe/fe-core/src/main/java/org/apache/doris/catalog/FunctionSet.java

```diff
@@ -2336,6 +2336,10 @@ private void initAggregateBuiltins() {
                     prefix + OFFSET_FN_UPDATE_SYMBOL.get(t),
                     null, null, null, true));
 
+            addBuiltin(AggregateFunction.createAnalyticBuiltin(
+                     "nth_value", Lists.newArrayList(t, Type.BIGINT, t), t, t,
```

Comment:
It seems that MySQL has no default value and directly returns null

Reply:
Default value have been removed

Reply:
> It seems that MySQL has no default value and directly returns null

Default value argument have been removed